frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -24,6 +34,11 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
 }) => {
   const { t } = useTranslation();
   const [handlePromise, inProgress, errorMessage] = usePromiseHandler();
+  const [operands, setOperands] = React.useState<K8sResourceKind[]>([]);
+  const [operandsLoaded, setOperandsLoaded] = React.useState<boolean>(false);
+  const [loadOperandsErrorMessage, setLoadOperandsErrorMessage] = React.useState<string>('');
```

Comment:
We should consider making a `useOperands` hook that handles all of this for us so the logic can be reused. Some examples of similar hooks here:

https://github.com/openshift/console/blob/d12d601f8becd17d61df7c17271004a7d1f04b4f/frontend/packages/console-shared/src/hooks/version.ts

Reply:
Added `useOperandsWatcher`